docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,62 @@
+---
+title: .NET API Analyzer
+description: Learn how the .NET API Analyzer can help detecting deprecated APIs and platform compatibility issues.
+author: oliag
+ms.author: mairaw
+ms.date: 12/11/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+---
+
+# .NET API Analyzer
+
+The .NET API Analyzer is a Roslyn analyzer that discovers potential compatibility risks for APIs with different platforms and detects calls to deprecated APIs. It can be useful for any developers of the .NET family at any stage of development.
+
+API Analyzer comes as [NuGet package](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) and after you reference it in a project, it automatically monitors the code and indicates problematic API usage. You can also get suggestions on possible fixes by clicking on the light bulb.The drop-down menu includes an option to suppress the warnings.
+
+## Discovering deprecated APIs
+When a deprecated API (for example `WebClient`)is used in a code, API Analyzer highlights it with a green squiggly line and shows a light bulb on the left
+
+![](media/api-analyzer/green-squiggle.jpg) !
+
+The Error List window contains warnings with a unique ID per deprecated API (in our example `DE004`). By clicking on the ID, you go to a webpage with detailed information about why the API was deprecated and what alternative APIs should be used.
```

Comment:
don't use we or our unless it's Microsoft recommending something. So change "our example" to something like "in the following example"